README.md

```diff
@@ -0,0 +1,70 @@
+# OFMT is a formatting and linting tool
+
+* **Formatting** is done solely by prettier. It is completely automated, and not configurable in any way other than changing the [`.prettierrc`](https://github.com/quesabe/ofmt/blob/add-readme/package.json) of this repo, and republishing the package.
+* **Linting** is not related to formatting in any way. The goal of linting is to catch early code quality issues, and enforce usage of best practices.
+
+## Install
+
+```shell
+npm install @ottofeller/ofmt --save-dev
+```
+
+## Configs
+
+The project contains the following configs:
+- `.prettierrc` formatting rules
+- `eslint.quality.cjs` with JS/TS-specific code quality and best practices rules
+- `eslint.tailwind.cjs` same as above, CSS-specific
+
+Direct use of configs is meant only for IDE support. For CLI runs use the provided executables.
+
+## Executables
+
+There are two _bin_ files that perform the code checking:
+- `ofmt` - installer/prettier formatter
+- `olint` - eslint formatter
+
+### ofmt
+
+#### Options:
+- `-l` - a flag to perform checking only. Without the flag prettier will rewrite files with fixed formatting.
+- _non-flag_ - the prettier target files. This can contain any of file paths, directory paths, and glob patterns.
+- `install` - a non-flag argument that creates a symlink to prettier config file and adds/extends eslint config within _package.json_. The second argument is the destination path.\
```

Comment:
@quesabe let's move this to the Installation section as a separate step. In the Options section I wouldn't actually keep anything other than `-l, --lint` flag. The fact that calling `ofmt` without a flag does the formatting should be clear from the context, and if not — should be clear from the examples below.

Reply:
Same for `olint` below. I don't think the usage guide should be kept in the Options section.